color/p5js/Motion-Figure:Ground/index.md

```diff
@@ -0,0 +1,9 @@
+layout: p5js
+author: Edmundo Mejía Galindo
+author-url: https://github.com/edgalindo/
+title: Landscape waves
+tags: [P5.js, Law of Multistable-perception, Law of Relativity of Properties, Law of Emotion, data, form, grid, motion]
+permalink: /motion-figure/ground/
```

Comment:
there is no folder motion-figure please use just motion as category permalink should be 

    /motion/landscape-waves/

folder should be like this as well.